.github/workflows/on_push.yaml

```diff
@@ -0,0 +1,20 @@
+name: On Push
+
+on:
+  push:
+    branches:
+    - main
+    - track/**
+
+jobs:
+
+  on-push:
+    name: Get ROCKs modified and build-scan-test-publish them
+    uses: canonical/charmed-kubeflow-workflows/.github/workflows/get-rocks-modified-and-build-scan-test-publish.yaml@main
+    permissions:
+      pull-requests: read
+    secrets: inherit
+    with:
+      microk8s-channel: 1.24/stable
+      juju-channel: 2.9/stable
```

Comment:
Why not use newer versions of MicroK8s and juju here?

Reply:
the version of this rock is still `ckf-1.7`
so these are the corresponding uk8s and juju

Reply:
Ahh good point!